### ISSUE TITLE
Mismatch between documentation and diagram for FixedValueMap?

### ISSUE THREAD

**Issue by sosna** [sdmx-twg/sdmx-im#30]
The documentation for `FixedValueMap` (p. 114) provides the following description: "Links a Component (source or target) to a fixed value.".

However, the Relationship diagram on page 111 has the following cardinality between `FixedValueMap` and `Component`: `0..*`. Should it not be `1` instead?

The XML schemas are in line with the description (i.e. only one component is referenced):

```xml
<xs:complexType name="FixedValueMapType">
	<xs:annotation>
		<xs:documentation>FixedValueMapType defines the structure for providing a fixed value for a source or target component.</xs:documentation>
	</xs:annotation>
	<xs:complexContent>
		<xs:extension base="common:AnnotableType">
			<xs:sequence>
				<xs:choice>
					<xs:element name="Source" type="common:IDType"/>
					<xs:element name="Target" type="common:IDType"/>
				</xs:choice>
				<xs:element name="Value" type="xs:string" minOccurs="0" maxOccurs="unbounded">
					<xs:annotation>
						<xs:documentation>The fixed value for the component.</xs:documentation>
					</xs:annotation>
				</xs:element>
			</xs:sequence>
		</xs:extension>
	</xs:complexContent>
</xs:complexType>
```

Thanks!

**Comment by stratosn** [comment]
Indeed, the relationship seems wrong, as one Component should be mapped in each `FixedValueMap`.
This is a bug in the IM.

In addition, it looks like the schema needs also some attention, as it now allows `0` to `unbounded` values, where there should be only one fixed value.
@dosse does this makes sense? 

I also see that in the [json-schema](https://github.com/sdmx-twg/sdmx-json/blob/71fe5eaa9fcd29e3c15f2f0216a19b9b650b1dbd/structure-message/tools/schemas/2.0.0/sdmx-json-structure-schema.json#L5483) it is allowed to have `1..*` values.
